procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        tmp = await latiss.rem.atarchiver.evt_imageInOODS.next(flush=False, timeout=10)
</pre><p> </p><p>Instead of consider a better name for <code>tmp</code>. Maybe <code>image_in_oods = await ...<span class="ql-cursor">﻿</span></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>